packages/grid/src/vaadin-grid.js

```diff
@@ -671,13 +671,15 @@ class Grid extends ElementMixin(
     cell.id = slotName.replace('-content-', '-');
     cell.setAttribute('tabindex', '-1');
     cell.setAttribute('role', tagName === 'td' ? 'gridcell' : 'columnheader');
-    cell.addEventListener('mouseenter', (event) => {
+
+    // Use pointer events to avoid tooltip flash on iOS
+    cell.addEventListener('pointerenter', (event) => {
```

Comment:
Seems to fix the issue on iOS. However, this removes the ability to get the tooltip to show on Android (by long pressing a cell).

If the intention is that the tooltips should not show on pointer actions at all on touch devices, should we rather just skip adding any event listeners here in case `isIOS` or `isAndroid`?

Reply:
Right, we can do that for now. I will update the PR.

Reply:
Done. Changed to use `isAndroid` and `isIOS` explicitly.